utils/payment-actions.js

```diff
@@ -147,3 +147,9 @@ export function completePurchase() {
   )
   cy.waitAndGet('.payment-submit-wrap > button.submit:visible', 3000).click()
 }
+
+export function insertCoupon() {
+  cy.get('#cart-link-coupon-add').click({ force: true })
```

Comment:
Do you need `force: true`?

Reply:
Yes, I do. For some reason, I can't manage to properly access it without the `force: true`. I think it has something to do with those elements being wrapped by another elements.